tRecorderApi/api/file_transfer/FileUtility.py

```diff
@@ -212,3 +218,83 @@ def compile_into_tr(self, root_dir):
     def check_if_path_exists(path):
         path_exist = os.path.exists(path)
         return path_exist
+
+    def __cleanup_files(self, cleanup_type):
+        fs_records = []
+        db_records = []
+
+        if cleanup_type == CleanupType.TAKE:
```

Comment:
these conditional statements reuse a lot of code that can probably be refactored into a function (you can pass a class in as a variable I believe, in order to handle the orm call if you want).